src/pages/components/SecondaryBar.qml

```diff
@@ -49,7 +49,14 @@ Item {
         // Spacer for pushing Search, Favorite, Share, Downloads to the right hand side
         Item {
             height: parent.height
-            width: parent.width - addTabButton.width - forwardButton.width - midIconWidth * 3 - downloadsButton.width
+            width: parent.width - addTabButton.width - forwardButton.width - midIconWidth * 4 - downloadsButton.width
+        }
+
+        Browser.IconButton {
+            width: midIconWidth
+            icon.source: "image://theme/icon-m-file-download-as-pdf"
+            active: webView.contentItem && !webView.loading
```

Comment:
active: webView.contentItem &&  webView.contentItem.active && !webView.contentItem.loading

I'd assume that contentItem must be active. Maybe loading guard could be ignored -- just wondering case where loading doesn't complete or/and there's already content in the view.